### DISCUSSION TITLE
New Buttons for Toolbar?

### DISCUSSION THREAD

**Discussion by jaydeflix** [SubtitleEdit/subtitleedit#5716] [Ideas]
The options for buttons to add to the toolbar are a little limited. What are the thoughts on making any menu item addable to the toolbar?

Scenario: I use Subtitle Edit to generate subs (duh) for social media. Twitter and FB both let me upload an SRT, but LinkedIn requires burnt-in subs[1]. I always forget SubtitleEdit can do it for me and use HandBrake to build the video with burnt-in subs. I'd rather just do it all in one tool and yes, I'm the key problem here, it'd be helpful if the Video-Generate video with burned-in subs was available as a toolbar button. But, why limit it to 'Just add this one button as an option' when 'Any menu option should be available' (tho I recognize that'd be a much more major overhaul of the settings page.)

[1] Unless you do it natively. There's a Beta API that allows for a separate CC file, but there's no ETA for a release of that update, and then waiting for the app devs to get that rolled in.

**Comment by niksedk** [comment]
I could add the "burn in" toolbar button.. if I had a nice icon ;)

**Reply by jaydeflix** [reply]
Do I have to do all the work? ;-)

Seriously tho, give me a few days. But let me be lazy: What's the file specs you need?

**Reply by LeonCheung** [reply]
@niksedk I'd like to try it if you will. :)

**Comment by niksedk** [comment]
Thx for the nice icon @LeonCheung :)
Beta updated: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.4/SubtitleEditBeta.zip

**Comment by niksedk** [comment]
![image](https://user-images.githubusercontent.com/1261331/150563380-e366b601-c509-4926-8845-6330ad5f0ab3.png)
